UnitySDK/Assets/ML-Agents/Scripts/LearningBrain.cs

```diff
@@ -135,9 +136,9 @@ protected override void DecideAction()
 #else
             if (agentInfos.Count > 0)
             {
-                throw new UnityAgentsException(string.Format(
-                    @"The brain {0} was set to inference mode but the Tensorflow 
-                        library is not present in the Unity project.",
+                Debug.LogError(string.Format(
```

Comment:
Why this change?

Reply:
If an error is thrown, the rest of the method is not called which causes a cascade of errors that made this exception hard to find. The quickest fix I could find was to log the error instead.